pallets/baby-liminal/src/lib.rs

```diff
@@ -18,9 +21,12 @@ const STORAGE_VERSION: StorageVersion = StorageVersion::new(0);
 
 /// We store verification keys under short identifiers.
 pub type VerificationKeyIdentifier = [u8; 4];
```

Comment:
up to you, but we may want to take this opportunity and increase length a bit (coming up with 4-letter shortcuts is not always comfy)

Reply:
ok